openmetadata-ui/src/main/resources/ui/src/components/Settings/Applications/AppDetails/ApplicationsClassBase.ts

```diff
@@ -34,6 +34,16 @@ class ApplicationsClassBase {
     return null;
   }
 
+  public getFloatingApplicationEntityList(): string[] {
+    return [];
+  }
```

Comment:
Do we need this function as well?

Reply:
yes, this is for having a check in the OSS as well and can be extends in Collate to add any other elements with same behaviour if introduced!